mantle/cmd/ore/gcloud/promote-image.go

```diff
@@ -104,6 +104,9 @@ func runPromoteImage(cmd *cobra.Command, args []string) {
 		// Perform the deprecation if the image is not already deprecated.
 		// We detect if it is active by checking if it either doesn't
 		// have any deprecation state or if it is explicitly ACTIVE.
+		//nolint // Add nolint to pass golang-ci lint,
+		// drop it when the error is gone, it's already fixed
```

Comment:
Optional nit: this comment doesn't say what the problem is.  I had to follow the issue link to see what was going on.

Reply:
Heh, that comment is copied from an earlier site a few lines higher up. I added a separate commit that enhances the comment.